readme/the-roadmap.md

```diff
@@ -0,0 +1,15 @@
+# Roadmap
+
+Here is what The Monaco Protocol Foundation is currently working on from both a technical and foundation point of view.
+
+# Monaco Protocol Foundation
+
+- Publish whitepaper
+- Form official foundation entity 
+
+# Technical
```

Comment:
Complete **security** audit of the protocol. 
***Further** open source the monaco protocol
Allow code contributions to the SDK and protocol